components/BlockLocationListItem.vue

```diff
@@ -0,0 +1,368 @@
+<template lang="html">
+    <div :class="classes">
+        <div class="card-container">
+            <responsive-image
+                v-if="image"
+                class="image"
+                :image="image"
+            />
+            <div
+                v-else
+                :class="['image affiliate']"
+            />
+
+            <div
+                class="library"
+            >
+                <smart-link
+                    v-if="to"
+                    :to="to"
+                    class="title"
+                >
+                    {{ title }}
+                </smart-link>
+                <div class="text">
+                    <div
+                        v-if="day || hour"
+                        class="time"
+                    >
+                        <SvgIconClock />
+                        <span>{{ day }}</span>
+                        <div class="hour">
+                            <span>{{ hour }}</span>
+                        </div>
+                    </div>
+                    <icon-with-link 
+                        v-if="reserveSeat"
+                        text="Reserve a Seat"
+                        icon-name="svg-icon-calendar"
+                        :to="reserveSeat"
+                        class="reserve"
```

Comment:
needs to be clickable to direct to its own link. right now, clicking here goes to the same link as the parent. same for `icon-with-link location` selector